imgui.go

```diff
@@ -686,3 +686,62 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// BeginColumns calls BeginColumnsV().
+func BeginColumns(count int, label string) {
+	BeginColumnsV(count, label, ColumnsFlagsNone)
+}
+
+// BeginColumnsV creates a column layout of the specified number of columns.
+func BeginColumnsV(count int, label string, flags int32) {
+	labelArg, labelFin := wrapString(label)
+	defer labelFin()
+	C.iggBeginColumns(C.int(count), labelArg, C.int(flags))
+}
+
+// NextColumn next column, defaults to current row or next row if the current row is finished.
+func NextColumn() {
+	C.iggNextColumn()
+}
+
+// GetColumnIndex get current column index.
+func GetColumnIndex() int {
```

Comment:
Applies to this `Get*` function any further: In Go, getters are not prefixed with `Get`. Please remove.